meta-mel/recipes-support/consolekit/consolekit/add-dependency-on-var-volatile.mount-for-shutdown.patch

```diff
@@ -0,0 +1,184 @@
+ data/console-kit-log-system-restart.service.in  |  9 -------
+ data/console-kit-log-system-shutdown.service.in | 10 +++++++
+ data/console-kit-log-system-stop.service.in     |  9 -------
+ data/console-kit-shutdown-handler               | 26 ++++++++++++++++++
+ 5 files changed, 45 insertions(+), 45 deletions(-)
+ delete mode 100644 data/console-kit-log-system-restart.service.in
+ create mode 100644 data/console-kit-log-system-shutdown.service.in
+ delete mode 100644 data/console-kit-log-system-stop.service.in
+ create mode 100644 data/console-kit-shutdown-handler
+
+diff --git a/data/Makefile.am b/data/Makefile.am
+index 248f7cd..855a223 100644
+--- a/data/Makefile.am
++++ b/data/Makefile.am
+@@ -24,8 +24,7 @@ if HAVE_SYSTEMD
+ systemdsystemunit_DATA = \
+ 	console-kit-daemon.service \
+ 	console-kit-log-system-start.service \
+-	console-kit-log-system-stop.service \
+-	console-kit-log-system-restart.service
++	console-kit-log-system-shutdown.service
+ 
+ console-kit-daemon.service: console-kit-daemon.service.in
+ 	$(edit) $< >$@
+@@ -33,34 +32,18 @@ console-kit-daemon.service: console-kit-daemon.service.in
+ console-kit-log-system-start.service: console-kit-log-system-start.service.in
+ 	$(edit) $< >$@
+ 
+-console-kit-log-system-stop.service: console-kit-log-system-stop.service.in
+-	$(edit) $< >$@
+-
+-console-kit-log-system-restart.service: console-kit-log-system-restart.service.in
++console-kit-log-system-shutdown.service: console-kit-log-system-shutdown.service.in
+ 	$(edit) $< >$@
+ 
+ install-data-hook:
+ 	$(MKDIR_P) -m 755 \
+-		$(DESTDIR)$(systemdsystemunitdir)/basic.target.wants \
+-		$(DESTDIR)$(systemdsystemunitdir)/halt.target.wants \
+-		$(DESTDIR)$(systemdsystemunitdir)/poweroff.target.wants \
+-		$(DESTDIR)$(systemdsystemunitdir)/reboot.target.wants \
+-		$(DESTDIR)$(systemdsystemunitdir)/kexec.target.wants
++		$(DESTDIR)$(systemdsystemunitdir)/basic.target.wants
+ 	( cd $(DESTDIR)$(systemdsystemunitdir)/basic.target.wants && \
+ 		rm -f console-kit-log-system-start.service && \
+ 		$(LN_S) ../console-kit-log-system-start.service )
+-	( cd $(DESTDIR)$(systemdsystemunitdir)/halt.target.wants && \
+-		rm -f console-kit-log-system-stop.service && \
+-		$(LN_S) ../console-kit-log-system-stop.service )
+-	( cd $(DESTDIR)$(systemdsystemunitdir)/poweroff.target.wants && \
+-		rm -f console-kit-log-system-stop.service && \
+-		$(LN_S) ../console-kit-log-system-stop.service )
+-	( cd $(DESTDIR)$(systemdsystemunitdir)/reboot.target.wants && \
+-		rm -f console-kit-log-system-restart.service && \
+-		$(LN_S) ../console-kit-log-system-restart.service )
+-	( cd $(DESTDIR)$(systemdsystemunitdir)/kexec.target.wants && \
+-		rm -f console-kit-log-system-restart.service && \
+-		$(LN_S) ../console-kit-log-system-restart.service )
++	( cd $(DESTDIR)$(systemdsystemunitdir)/basic.target.wants && \
++		rm -f console-kit-log-system-shutdown.service && \
++		$(LN_S) ../console-kit-log-system-shutdown.service )
```

Comment:
@bboccoqq, wouldn't it be more appropriate to use the Install section in the service so it can be enabled/disabled using systemctl on the target? This is more like
[Install]
WantedBy="basic.target"

Reply:
This is the way the package is written, I did not try to change that behaviour. I don't think there is an expectation that the service is to be enabled/disabled at runtime.

Reply:
Considering that the expectation is not to have something that is runtime configurable through systemctl I believe this is fine.